scripts/test/lld.py

```diff
@@ -36,8 +39,11 @@ def test_wasm_emscripten_finalize():
       if ext != '.out' and not os.path.exists(expected_file):
         continue
 
-      cmd = (WASM_EMSCRIPTEN_FINALIZE +
-             [wast_path, '-S', '--global-base=568', '--initial-stack-pointer=16384'] + ext_args)
+      cmd = WASM_EMSCRIPTEN_FINALIZE + [wast_path, '-S'] + ext_args
+      if 'shared' in os.path.basename(wast_path):
```

Comment:
Nit: 2x repetition of 4lines each, maybe extract this all to a function, and call it like `cmd += args_for_wast(wast_path)`

Reply:
Related nit: `args` is an iffy name for a global variable. Can hide them inside the function if you do extract it